app/src/main/webapp/WEB-INF/jsps/tiles/head.jsp

```diff
@@ -5,7 +5,7 @@ You can override it with your own file via WEB-INF/tiles-def.xml
 
 <%@ include file="/WEB-INF/jsps/taglibs-struts2.jsp" %>
 
-<script src="<s:url value='/webjars/jquery/3.6.4/jquery.min.js' />"></script>
+<script src="<s:url value='/webjars/jquery/3.7.1/jquery.min.js' />"></script>
```

Comment:
@snoopdave head.jsp is already updated.

Reply:
don't know how I missed that! 

Merge whenver you are ready and I'll do some manual UI testing to make sure jQuery is working and my PR does not break things.